packages/site/src/layout/Layout.tsx

```diff
@@ -84,7 +88,7 @@ export const LayoutWrapper = ({ children }: PropsWithChildren) => {
     <div
       style={{
         display: "flex",
-        background: "var(--color-surface)",
+        background: "var(--color-surface--background)",
```

Comment:
Site background is now `surface--background`, so we can remove the explicit color from the sidenav and siderail. We now apply `surface` to the main content of the page as an implicit elevation tactic.